pkg/cvo/updatepayload.go

```diff
@@ -189,13 +190,40 @@ func (r *payloadRetriever) fetchUpdatePayloadToDir(ctx context.Context, dir stri
 		},
 	}
 
+	// Prune older jobs while gracefully ignoring errors.
+	r.pruneJobs()
```

Comment:
Question:
would be want the function to return a TooManyJobs error type so that we can fail hard when we see that error because that would be indicator of an actual error?